refactor_me.py

```diff
@@ -1,26 +1,35 @@
 #!/usr/bin/env python3
 
+__all__ = ['Expense', 'sum_expenses', 'print_expenses']
```

Comment:
can we use tuple here?

Reply:
sure, I will. i think `tuple` would be more preferable here